html/index.html

```diff
@@ -2,15 +2,20 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Well, I just think it's very specific.
We should do that if we aim to support every browser.
I'm almost certain every developer here uses Chrome of Firefox, and we don't have any intention of going live right now.